### PR TITLE
Add some enhancement for Hyper-V testing

### PR DESCRIPTION
**Pull request by lubaihua33**

### CODE REVIEW

**Review by squirrelsc — DISMISSED** [review]

File: Libraries/IntegrationServiceLibrary.psm1 [hunk 1]
```diff
@@ -741,13 +741,13 @@ Function New-BackupSetup {
 		[String] $HvServer
 	)
 	# Install the Windows Server Backup
-	$winFeatureName = "Windows-Server-Backup"
-	if ((Get-WindowsFeature $winFeatureName).InstallState -ne "Installed") {
-		Write-LogInfo "Installing the $winFeatureName ..."
-		if ( (Add-WindowsFeature $winFeatureName).Success -eq "True" ) {
-			Write-LogInfo "Install the $winFeatureName successfully"
+	$winFeatureName = "WindowsServerBackup"
+	if ((Get-WindowsOptionalFeature -Online -FeatureName $winFeatureName).State -ne "Enabled") {
```
Comment (squirrelsc) [inline Libraries/IntegrationServiceLibrary.psm1:745]: Make sure it works with windows 10 server also.

Comment (lubaihua33) [reply Libraries/IntegrationServiceLibrary.psm1:745]: Yes, the feature "WindowsServerBackup" should be in the Windows server system. I confirmed in Server 2019 it worked. In Windows 10, there is no "WindowsServerBackup" feature.

Comment (squirrelsc) [reply Libraries/IntegrationServiceLibrary.psm1:745]: How about `Get-WindowsFeature to `Get-WindowsOptionalFeature`? Does the `Get-WindowsOptionalFeature` supports windows 10 servers?

**Review by squirrelsc — DISMISSED** [review]

File: Libraries/IntegrationServiceLibrary.psm1 (hunk 1 above)
Comment (lubaihua33) [reply Libraries/IntegrationServiceLibrary.psm1:745]: I don't think there is a windows 10 server, you mean windows server 2019? I tried in server 2019, it supports.

**Review by squirrelsc — APPROVED** [review]

**Pull request merged**